config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'users#index'
+  get 'users/new'
```

Comment:
This is unneeded. The line below defines all of your RESTful routes for your user model.

To check which routes you have defined in your app you can run `bundle exec rake routes`